src/Message/Response.php

```diff
@@ -388,6 +389,7 @@ public function getNextChargeDate()
 
     /**
      * @return array<Transaction>|null
+     * @psalm-suppress PossiblyInvalidArrayOffset
```

Comment:
which one is possibly invalid?

Reply:
foreach ($this->data['data'] as $row)  on l.403 -- it thinks data['data'] is possibly invalid, despite the test for whether or not data['data'] is set a few lines earlier.